pkg/cmd/tknpac/bootstrap/bootstrap.go

```diff
@@ -183,6 +184,7 @@ func GithubApp(run *params.Run, ioStreams *cli.IOStreams) *cobra.Command {
 }
 
 func addGithubAppFlag(cmd *cobra.Command, opts *bootstrapOpts) {
+	cmd.PersistentFlags().StringVar(&opts.GithubOrganizationName, "github-organization-name", "", "Wether you want to target an organization instead of the current user")
```

Comment:
```suggestion
	cmd.PersistentFlags().StringVar(&opts.GithubOrganizationName, "github-organization-name", "", "Whether you want to target an organization instead of the current user")
```